diff -Naur -x '*.orig' -x '*~' nss_ldap-265/Makefile.am nss_ldap-265-Makefile/Makefile.am
--- nss_ldap-265/Makefile.am	2009-11-06 11:28:08.000000000 +0100
+++ nss_ldap-265-Makefile/Makefile.am	2009-11-06 15:05:18.000000000 +0100
@@ -25,7 +25,7 @@
 	dnsconfig.c irs-nss.c pagectrl.c ldap-sldap.c ldap-init-krb5-cache.c \
 	vers.c
 
-nss_ldap_so_LDFLAGS = @nss_ldap_so_LDFLAGS@
+nss_ldap_so_LDFLAGS = @nss_ldap_so_LDFLAGS@ $(VERSION_SCRIPT_FLAGS)
 
 NSS_LDAP_PATH_CONF = @NSS_LDAP_PATH_CONF@
 NSS_LDAP_PATH_ROOTPASSWD = @NSS_LDAP_PATH_ROOTPASSWD@
@@ -109,5 +109,5 @@
 	@$(NORMAL_UNINSTALL)
 
 vers.c: $(top_srcdir)/CVSVersionInfo.txt
-	CVSVERSIONDIR=$(top_srcdir) vers_string -v
+	CVSVERSIONDIR=$(top_srcdir) $(top_srcdir)/vers_string -v
 
